docs/platform/porting/graphics-and-ui.md

```diff
@@ -36,169 +36,192 @@ lrwxrwxrwx  1 root root    20 Jul 28  2016 libtbm_sprd.so.0 -> libtbm_sprd.so.0.
+The following table lists the `bufmgr` backend interface functions of `tbm_backend_module`. For more information, see [tbm_backend.h](https://review.tizen.org/gerrit/gitweb?p=platform/core/uifw/libtbm.git;a=tree;h=refs/heads/tizen;hb=refs/heads/tizen):
 
-**Table: tbm_bo functions**
+**Table: bufmgr functions**
 
-| Function          | Description                              | Mandatory                                         |
+| Function          | Description                              | Mandatory                                |
 | ----------------- | ---------------------------------------- | ---------------------------------------- |
-| `bo_alloc()`      | Allocates the buffer object. If the backend wants to reuse the `bo` private at frontend, return the same pointer of the `bo` private. | Yes                                |
-| `bo_free()`       | Frees the buffer object. The frontend calls this function when it does not use the `bo` private. | Yes                                |
-| `bo_import()`     | Imports the buffer object associated with the key. If the backend does not support buffer sharing by the TBM key, the function pointer must be set to `NULL`. | No                                 |
-| `bo_export()`     | Exports the buffer object. If the backend does not support buffer sharing by TBM key, the function pointer must be set to `NULL`. | No                                 |
-| `bo_import_fd()`  | Imports the buffer object associated with the prime `fd`. The `tbm_fd` must be freed by the user. If the backend does not support buffer sharing by TBM `fd`, the function pointer must be set to `NULL`. | Yes (Must support buffer sharing by TBM `fd`.) |
-| `bo_export_fd()`  | Imports the buffer object associated with the prime `fd`. The `tbm_fd` must be freed by the user. If the backend does not support buffer sharing by TBM `fd`, the function pointer must be set to `NULL`. | Yes (Must support buffer sharing by TBM `fd`.) |
-| `bo_get_flags()`  | Gets the TBM flags of memory type.        | Yes                                |
-| `bo_size()`       | Gets the size of a buffer object.        | Yes                                |
-| `bo_get_handle()` | Gets the `tbm_bo_handle` according to the device type. | Yes                                |
-| `bo_map()`        | Maps the buffer object according to the device type and the option. | Yes                                |
-| `bo_unmap()`      | Unmaps the buffer object.                | Yes                                |
-| `bo_lock()`       | Locks the buffer object with a device and an opt. | No                                 |
-| `bo_unlock()`     | Unlocks the buffer object.               | No                                 |
-
-The following table lists the TBM backend interface functions for `tbm_surface`.
-
-**Table: tbm_surface functions**
+| `bufmgr_get_capabilities()`           | Get the capabilities of a buffer manager. | Yes |
```

Comment:
@Reni-90 In this page, the descriptions in the tables are in the third person singular. 
(Does ... /Supports ... /Gets ... /Sets ....)
I think it is reasonable that we can consider the "Function" names as the subject of the Description.

However, in this table, "**Table: bufmgr functions**", "**Table: bo functions**", and the first row of the "**Table: Layer backend interface functions**" table (`layer_get_capability()`), 
the descriptions are not starting with the 3rd person sigular.

To get the consistency through the page, how about using the 3rd person singular to there also?

**Example)**
Gets the capabilities of a buffer manager.
Sets (bind) the native display. If the backend needs to get the native display, use this backend function.
Gets the format list and the number to be supported by the backend.
...

Reply:
I agree with you.

Reply:
I will